providers/gcp/resources/discovery/gcr/resolver_gcr.go

```diff
@@ -20,18 +18,19 @@ func (r *GcrResolver) Name() string {
 }
 
 func (r *GcrResolver) AvailableDiscoveryTargets() []string {
-	return []string{common.DiscoveryAuto, common.DiscoveryAll}
+	return []string{"auto", "all"}
 }
 
-func (r *GcrResolver) Resolve(ctx context.Context, root *asset.Asset, t *providers.Config, credsResolver vault.Resolver, sfn common.QuerySecretFn, userIdDetectors ...providers.PlatformIdDetector) ([]*asset.Asset, error) {
-	resolved := []*asset.Asset{}
-	repository := t.Host
+// func (r *GcrResolver) Resolve(ctx context.Context, root *inventory.Asset, t *inventory.Config, credsResolver vault.Resolver, sfn common.QuerySecretFn, userIdDetectors ...providers.PlatformIdDetector) ([]*inventory.Asset, error) {
+func (r *GcrResolver) Resolve(ctx context.Context, root *inventory.Asset, conf *inventory.Config, credsResolver vault.Resolver) ([]*inventory.Asset, error) {
+	resolved := []*inventory.Asset{}
+	repository := conf.Host
 
```

Comment:
i guerss this code can go in `discovery.go`

Reply:
Done